### DISCUSSION TITLE
Add zsh suppourt (macOS)

### DISCUSSION THREAD

**Discussion by Shad0wSeven** [GitSquared/edex-ui#1010] [Questions] [answered]
The default terminal seems to be bash, I don't think much  has to be said, but when i tried running `zsh`, the zsh worked, but the rest of the UI (specifically the filesystem view) for instance, did not respond to changes made in zsh, so file picker would not respond to up a directory or down a directory, and issued bash commands when interacting with the GUI, could there be some sort of detection, or anything to allow these two to integrate, as 90% of people i feel prefer zsh over bash, and even just using edex as a novelty, commands that are reflex at this point such as `../` or using custom plugins would be really nice.

Also as of macOS Catalina, zsh is default, so having full support would be awesome!

*I may also be absolutely stupid and missing a config file or something*

**Comment by GitSquared** [comment, marked as answer]
eDEX tracks the directory changes of a specific process, which is why when you start zsh from bash, it is still tracking the bash process and not your activity and zsh.

However, you can indeed change the default shell! Pop up the settings with ctrl+shift+s and change `bash` to `zsh` in the `shell` setting. Hit save to disk, restart edex, and you should be good.
